k8s/pod.yml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: sample-app
+  namespace: default
+spec:
+  initContainers:
+    - name: node-init
+      image: bengl/test-auto-inst:latest  # put the library image here
+      imagePullPolicy: Never
+      command: ["sh", "copy-lib.sh", "/autoinstrumentation"]
+      volumeMounts:
+        - name: autoinstrumentation
+          mountPath: /autoinstrumentation
+  containers:
+    - name: app
+      image: bengl/test-sample-app # put the app image here
```

Comment:
There are multiple references pointing to `bengl/<something>` Is this OK? Or do we have to change it to something like `dd/test-sample-app`...

Reply:
It's fine. Mentioned in the readme.